play/src/pusher/services/MatrixProvider.ts

```diff
@@ -1,9 +1,42 @@
 import axios from "axios";
-import { MATRIX_API_URI, MATRIX_DOMAIN } from "../enums/EnvironmentVariable";
+import { EventType } from "matrix-js-sdk";
+import { MATRIX_ADMIN_USER, MATRIX_API_URI, MATRIX_DOMAIN } from "../enums/EnvironmentVariable";
 
+const ADMIN_CHAT_ID = `@${MATRIX_ADMIN_USER}:${MATRIX_DOMAIN}`;
```

Comment:
I think this is not needed.
We can always use our own Matrix server to create the rooms.
A client can use its own Matrix server for its users, but the rooms can stay on our server since Matrix is distributed.